content/includes/business-logos.html

```diff
@@ -0,0 +1,19 @@
+<section id="business-logo">
+    <div class="ui very padded container">
+        <div class="ui four column grid">
+            <div class="column">
+                <img class="ui small image" src="{{ SITE_URL }}/images/amd-logo.svg" alt="AMD Logo">
```

Comment:
This doesn't need capitalization here and instances below:

```suggestion
                <img class="ui small image" src="{{ SITE_URL }}/images/amd-logo.svg" alt="AMD logo">
```